airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../../../../../../scss/colors";
+
+.container {
+  width: 100%;
+
+  &.zeroBalance {
+    background-color: colors.$red-50;
```

Comment:
Could we maybe make this a property on `InfoBox` instead, like `style="error"` so that we can use this across the app, and make sure it's not a one-off styling here.

Reply:
Sounds great!